components/admin/requests/physician-assessment/Form.tsx

```diff
@@ -1,23 +1,13 @@
-import {
-  FormControl,
-  FormLabel,
-  Input,
-  Text,
-  Stack,
-  FormHelperText,
-  Box,
-  Divider,
-  RadioGroup,
-  Radio,
-  Textarea,
-} from '@chakra-ui/react'; // Chakra UI
-import { PatientCondition, PermitType } from '@lib/graphql/types';
+import { Text, Stack, FormHelperText, Box, Divider, Radio } from '@chakra-ui/react'; // Chakra UI
+import DateField from '@components/form/DateField';
+import RadioGroupField from '@components/form/RadioGroupField';
+import TextArea from '@components/form/TextAreaField';
+import TextField from '@components/form/TextField';
 import { PhysicianAssessment } from '@tools/admin/requests/physician-assessment';
-import { ChangeEventHandler } from 'react';
 
 type PhysicianAssessmentFormProps = {
   readonly physicianAssessment: PhysicianAssessment;
-  readonly onChange: (updatedData: PhysicianAssessment) => void;
+  readonly onChange?: (updatedData: PhysicianAssessment) => void;
```

Comment:
Can remove this prop if it's not being used anymore (also remove from `create-new.tsx`)